static/app/components/events/aiSuggestedSolution/suggestion.tsx

```diff
@@ -125,6 +118,7 @@ export function Suggestion({onHideSuggestion, projectSlug, event}: Props) {
       {
         query: {
           consent: suggestedSolutionLocalConfig.individualConsent ? 'yes' : undefined,
+          pii_certified: isStaff ? (piiCertified ? 'yes' : 'no') : undefined,
```

Comment:
When `piiCertified` updates I don't think it refetches the query right, you have to call `dataRefetch`

Reply:
better to just disable the query `enabled: false` when is staff and pii not certified?